test/testdata/packager/strict_dependencies/missing/__package.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+# typed: strict
+# enable-packager: true
+
+class Missing < PackageSpec
```

Comment:
I'd like to add an error for this case too, but there's a bunch of existing tests that need to updated to add the `strict_dependencies` line too, so I'll do it in a follow-up PR, to keep this PR small.